### ISSUE TITLE
Overlayed options cannot be swiped to on VoiceOver iOS

### ISSUE THREAD

**Issue by nickcolley** [alphagov/accessible-autocomplete#345]
This issue is from a May 2019 external accessibility audit report.

**WCAG Reference**:
- [1.3.1 Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
**Issue ID**: DAC_4

## Summary

After typing into the autocomplete, swiping to the options that appear is not possible.

When using the swipe gesture options cannot be accessed, this was an issue with displayMenu: 'overlay' turned on but might be an issue generally.

> "With VoiceOver for iPhone I was unable to swipe to the options once they appeared and needed to explore by touch to locate the available options which was an inconsistent method of navigation as I could not always locate them."

**Comment by nickcolley** [comment]
This seems to be related to the `role=combobox` being the wrapper for all elements, when I move `role=combobox` to the input element, this stops being an issue.

So I think https://github.com/alphagov/accessible-autocomplete/pull/355 will resolve this issue, but will leave this open until I can check for sure when the github page is deployed...

**Comment by nickcolley** [comment]
Confirmed this is fixed in the new release.

**Comment by nickcolley** [comment]
Confirmed as fixed in a re-test by DAC on 7th October 2019
